src/components/Calculator.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="container">
+      <div className="item item-1">0</div>
+      <div className="item item-2">AC</div>
+      <div className="item item-3">+/-</div>
+      <div className="item item-4">%</div>
+      <div className="item item-5 bg">÷</div>
+      <div className="item item-6">7</div>
+      <div className="item item-7">8</div>
+      <div className="item item-8">9</div>
+      <div className="item item-9 bg">x</div>
+      <div className="item item-10">4</div>
```

Comment:
As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/e404f92ae7b446ece2b5081c85779f0d9ab59731/math-magicians/project_components.md?plain=1#L40), kindly ensure that all components are class-based.